runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -269,6 +284,15 @@ public static CorfuRuntimeParametersBuilder builder() {
             private PriorityLevel priorityLevel = PriorityLevel.NORMAL;
             private Codec.Type codecType = Codec.Type.ZSTD;
             private MetricRegistry metricRegistry = null;
+            private MicroMeterRuntimeConfig microMeterRuntimeConfig =
+                    new MicroMeterRuntimeConfig(true,
+                    "CorfuMetrics", Duration.ofMinutes(1));
```

Comment:
Can you use the existing logger name https://github.com/CorfuDB/CorfuDB/blob/master/runtime/src/main/java/org/corfudb/util/MetricsUtils.java#L212  ?

We have put out documents that use that logger's name, using a different one might be more confusing.

Reply:
It would be nice to have a read me and explicitly spell it out too, that is, the configuration of metrics.

Reply:
done